js/util/helpers.js

```diff
@@ -866,6 +866,12 @@ define(["dao", "globals", "lib/knockout", "util/eventLog"], function (dao, g, ko
     function gameScore(arg) {
         return round(arg.pts + 0.4 * arg.fg - 0.7 * arg.fga - 0.4 * (arg.fta - arg.ft) + 0.7 * arg.orb + 0.3 * (arg.trb - arg.orb) + arg.stl + 0.7 * arg.ast + 0.7 * arg.blk - 0.4 * arg.pf - arg.tov, 1);
     }
+    
+    function plusMinus(arg, d) {
+	    if(typeof(d) === "undefined") { d = 1; }
+    	if(isNaN(arg)) { return ""; }
```

Comment:
Why would arg ever be NaN? Also, isNaN is [kind of a horrifying function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN). I would do something like `arg !== arg` to check for NaN, if it's really needed.